ArduPlane/quadplane.cpp

```diff
@@ -424,6 +424,12 @@ const AP_Param::GroupInfo QuadPlane::var_info2[] = {
     // @Range: 0 100
     AP_GROUPINFO("FWD_MANTHR_MAX", 20, QuadPlane, fwd_thr_max, 0),
 
+    // @Param: FWD_THR_MIN
+    // @DisplayName: VTOL minimum forward throttle percent
+    // @Description: Minimum value for manual forward throttle; allows for aft translation using backward tilt angles in tiltrotor configurations
+    // @Range: -100 0
+    AP_GROUPINFO("FWD_THR_MIN", 21, QuadPlane, fwd_thr_min, 0),
```

Comment:
21 was used by the old param `Q_TAILSIT_DSKLD`, so you have to use a new number down at the bottom of the table. Re-using numbers could mean that someone gets there old value of  `Q_TAILSIT_DSKLD` loaded instead of the default of 0.